.buildkite/packaging.pipeline.yml

```diff
@@ -10,6 +10,8 @@ env:
 
   PLATFORMS: "+all linux/amd64 windows/amd64 darwin/amd64 darwin/arm64"
   PLATFORMS_ARM: "+all linux/arm64"
+  PLATFORMS_FIPS: "+all linux/amd64"
```

Comment:
Since this is only `amd64`, WDYT about being more specific in the var name and the descriptions below?

Reply:
I followed the same naming as the non FIPS variables but we can be more explicit which is not a bad thing, will change it

Reply:
Done in df3d1077ccb3df096aa2c4d8c86cdac78bd517e6